src/cr/cube/matrix.py

```diff
@@ -714,6 +720,11 @@ def _row_generator(self):
             self._column_index,
         )
 
+    @lazyproperty
+    def _row_proportions(self):
+        """2D np.float64 ndarray of row proportions for each matrix cell."""
+        return (self._counts.T / self._rows_margin.T).T
```

Comment:
I don't think this is going to work when an MR dimension is involved. Pretty sure you're going to get a broadcast error when `self._rows_margin` is 2D, as it is for a CAT_X_MR slice:

```python
>>> import numpy as np
>>> counts = np.array([[[1,2],[2,3],[3,4]],[[4,5],[5,6],[6,7]]])
>>> counts
array([[[1, 2],
        [2, 3],
        [3, 4]],

       [[4, 5],
        [5, 6],
        [6, 7]]])
>>> rows_margin = np.sum(counts, axis=1)
>>> rows_margin
array([[ 6,  9],
       [15, 18]])
>>> (counts.T / rows_margin.T).T
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: operands could not be broadcast together with shapes (2,3,2) (2,2)
```

Also, when rows-margin is 1D, `rows_margin == rows_margin.T` so that transposition is spurious.

Reply:
CAT-X-MR has its own row_proportions definition:
```
    @lazyproperty
    def _row_proportions(self):
        """2D np.float64 ndarray of row proportions for each matrix cell."""
        return self._counts[:, :, 0] / np.sum(self._counts, axis=2)
```
line 1227

Reply:
yes that .T is pointless, REMOVED